guest-agents/qemu-guest-agent/pcre2/pkg.yaml

```
@@ -23,6 +23,13 @@ steps:
     install:
       - |
         make DESTDIR=/rootfs install
+    sbom:
+      outputPath: /rootfs/usr/local/share/spdx/pcre2.spdx.json
+      version: {{ .PCRE2_VERSION }}
+      cpes:
+        - cpe:2.3:a:pcre:pcre2:{{ .CONTAINER_TOOLKIT_VERSION }}:*:*:*:*:*:*:*
```

```suggestion
        - cpe:2.3:a:pcre:pcre2:{{ .PCRE2_VERSION }}:*:*:*:*:*:*:*
```

Also not sure if it should be built here, it's already being built in pkgs